main.py

```diff
@@ -2,7 +2,7 @@
 
 # ruff: noqa: I001
 
-from openg2p_spar_mapper_api.app import Initializer
+from src.openg2p_spar_mapper_api.app import Initializer
```

Comment:
remove `src` from the import statement. 
It should be:
`from openg2p_spar_mapper_api.app import Initializer`

You'll need to do `pip install -e .` from the root directory and run the app using `./main.py run`